packages/typespec-client-generator-core/test/types/multipart-types.test.ts

```diff
@@ -508,13 +510,15 @@ describe("typespec-client-generator-core: multipart types", () => {
     ) as SdkBodyModelPropertyType;
     ok(stringsOnePart);
     strictEqual(stringsOnePart.type.kind, "array");
+    strictEqual(stringsOnePart.type.valueType.kind, "string");
     ok(stringsOnePart.multipartOptions);
     strictEqual(stringsOnePart.multipartOptions.isMulti, false);
```

Comment:
nit: the option name `isMulti` seems a little confusing especially it is already in multipartOptions.

Reply:
In typespec/http, its name is `multi`. Since it is boolean, TCGC names it as `isMulti`

Reply:
I think since our final goal is to use some type to gradually consume these options, maybe it is fine to keep this name.